doc/releases/release-notes-3.2.rst

```diff
@@ -203,16 +203,29 @@ Boards & SoC Support
 * Added support for these SoC series:
 
   * renesas_smartbond da1469x SoC series
+  * GigaDevice GD32E50X
+  * GigaDevice GD32F470
 
 * Removed support for these SoC series:
 
 * Made these changes in other SoC series:
 
+  * gigadevice: Enable SEGGER RTT
+  * gd32f405: Fixed miss spelling config
+  * gd32vf103: Add missing include
+  * gd32vf103: Add :dtcompatible:`nuclei,systimer`, remove :dtcompatible:`riscv,machine-timer`
```

Comment:
this is probably not relevant enough to make into release notes

```suggestion
```

Reply:
Removed.